sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -232,14 +233,15 @@ public partial class TableSasBuilder
         public string RowKeyStart { get { throw null; } set { } }
         public System.DateTimeOffset StartsOn { get { throw null; } set { } }
         public string TableName { get { throw null; } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public string Version { get { throw null; } set { } }
```

Comment:
We should probably just delete the Version property.  The Storage folks decided they'd only support latest and we might as well follow suit unless we explicitly decide otherwise.

Reply:
I think I'll make it internal so that we can still set it from the latest version defined in `TableClientOptions`